docs/MVS_kpis_list.csv

```diff
@@ -1,14 +1,13 @@
 ref,label,:Unit:,:Valid Interval:,:Type:,category,see_also,:Definition:
 annual_total_flow,Aggregated flow,kWh,>=0,Numeric,technical,"average_flow, peak_flow","Dispatch of the asset over a year, aggregated generation, demand or throughput."
-annuity_om,"Annual operation, maintenance and dispatch expenses",currency/a,>=0,Numeric,economic,"annuity_total, costs_om_total","Annuity of the operation, maintenance and dispatch costs of the energy system, ie. Ballpoint number of the annual expenses for system operation."
-annuity_total,Annuity,currency/a,>=0,Numeric,economic,"annuity_om, costs_total",Annuity of the net present costs (NPC) of the energy system.
-attributed_costs,Costs attributed to the electricity sector,currency,>=0,Numeric,economic,costs_total,"Costs attributed to supplying the electricity sectors demand, based on Net Present Costs of the energy system and the share of electricity compared to the overall system demand."
-average_flow,Average flow,kWh,>=0,Numeric,technical,"annual_total_flow, peak_flow",Average value of the assets dispatch. Relation of peak dispatch and average dispatch indicates how much the asset is used in comparison to its actual installed capacity.
-automatic_report,Simulation report (html),None,None,html/pdf,files,,"Automatically generated simulation report, including the most important input data as well as all output data. The `html` be used interactively, while the `pdf` can be shared with partners."
-average_flow,Average flow,kWh,>=0,Numeric,technical,"annual_total_flow, peak_flow",Average value of the assets dispatch. Relation of peak dispatch and average dispatch indicates how much the asset is used compared to ist actual installed capacity.
-bar_chart_optimizedAddCap,Bar chart of optimal capacities,"kWh/kWp/kW,...",None,Figure,files,optimizedAddCap,A bar chart to compare the optimized additional capacities for each asset to be installed in the energy system. Please be aware that the units of the capacities may different.
+annuity_om,"Annual operation, maintenance and dispatch expenses",currency/a,>=0,Numeric,economic,"annuity_total, costs_om_total","Annuity of the operation, maintenance and dispatch costs of the energy system, i.e. ballpark number of the annual expenses for system operation."
+annuity_total,Annuity,currency/a,>=0,Numeric,economic,"annuity_om, costs_total",Annuity of the energy system's :ref:`net present costs (NPC) <costs_total>` .
```

Comment:
@Bachibouzouk but the `MVS_kpis_list.csv` does not use the "|", right? Shouldnt this be changed?

Reply:
inside the csv `,` should be used, I changed the code in `conf.py` already :)

Reply:
Ach, so hast du das gemeint! Dass im RTD "|" erscheinen soll. Jetzt hab ichs ;)